doc/en/conf.py

```diff
@@ -26,58 +9,23 @@
 from _pytest import __version__ as full_version
```

Comment:
Going to change this to import from `pytest`, no need to import from the private package.